requirements/base.txt

```diff
@@ -29,7 +29,7 @@ typing_extensions>=4.4.0,<5
 # NOTE: regex is not a direct dependency of SAM CLI, exclude version 2021.10.8 due to not working on M1 Mac - https://github.com/mrabarnett/mrab-regex/issues/399
 regex!=2021.10.8
 # NOTE: tzlocal is not a direct dependency of SAM CLI, but pin to 3.0 as 4.0 break appveyor jobs
-tzlocal==3.0
+tzlocal==5.0.1
```

Comment:
Above note mentions about 4.0 breaking appveyor jobs. We will not approve/merge this PR in favor of upcoming releases.